vmm/src/memory_manager.rs

```diff
@@ -1959,7 +1984,7 @@ impl MemoryManager {
                 .file_offset()
                 .unwrap()
                 .file();
-            memory_slot_fds.insert(slot, file.as_raw_fd());
+            memory_slot_fds.insert(slot.unwrap(), file.as_raw_fd());
```

Comment:
Please document why it is okay to unwrap here.

Reply:
Fixed